collections/e12c_motoway.yaml

```diff
@@ -20,9 +20,46 @@ License: |
  [GNU General Public License v3.0](https://github.com/hfisser/Truck_Detection_Sentinel2_COVID19/blob/master/LICENSE)
 Resources:
```

Comment:
Github doesn't allow me to comment on collapsed lines, but this collection has a mistake in the name I believe, (motoRway). Not connected to this PR, so fftr.

Reply:
Spoke with Dorothy and it's a typo and was fixed here.

Reply:
The folder name is still `e12c_motoway` btw. fftr

Reply:
True, missed that. Changed, thanks.